mm2src/coins/lp_coins.rs

```diff
@@ -328,6 +330,44 @@ impl From<CoinFindError> for RawTransactionError {
     }
 }
 
+#[derive(Debug, Display, Serialize, SerializeErrorType, Deserialize)]
+#[serde(tag = "error_type", content = "error_data")]
+pub enum GetMyAddressError {
+    CoinsConfCheckError(String),
+    CoinIsNotSupported(String),
+    #[display(fmt = "Internal error: {}", _0)]
+    Internal(String),
+    #[display(fmt = "Invalid request error error: {}", _0)]
+    InvalidRequest(String),
+    #[display(fmt = "Get Eth address error: {}", _0)]
+    GetEthAddressError(GetEthAddressError),
+}
+
+impl From<CryptoCtxError> for GetMyAddressError {
+    fn from(e: CryptoCtxError) -> Self { GetMyAddressError::Internal(e.to_string()) }
+}
+
+impl From<serde_json::Error> for GetMyAddressError {
+    fn from(e: serde_json::Error) -> Self { GetMyAddressError::InvalidRequest(e.to_string()) }
+}
```

Comment:
I guess all the manually impl of enums like this can be replaced with `EnumFromStringify`.
use case:
https://github.com/KomodoPlatform/atomicDEX-API/blob/e5d2109e735ee26640da89e37f72aad93806c85e/mm2src/coins/lp_coins.rs#L1894-L1901

Reply:
Thanks, added to `GetMyAddressError` and `GetNftInfoError`